docs/tutorial.rst

```diff
@@ -1256,7 +1256,7 @@ as aliases, by wrapping them in ``@()``.  For example:
 
     >>> @(_banana)
     'My spoon is tooo big!'
-    >>> echo "hello" | @(lambda args, stdin=None: stdin.strip() + args[0]) world
```

Comment:
Just wondering: Where did the space between "hello" and "world" and the EOL come from back then?